packages/lib-user/README.md

```diff
@@ -0,0 +1,49 @@
+# Zooniverse User and User Group Library
+
+A standalone library for the Zooniverse user stats, user group stats, and home pages.
```

Comment:
Suggestion to add a sentence here like "This library is intended to be imported into app-root as part of zooniverse.org..."